lib/hammer_cli_foreman_openscap/scap_content.rb

```diff
@@ -28,9 +28,20 @@ class BulkUpload < HammerCLIForemanOpenscap::CreateCommand
       success_message _('Scap Contents uploaded.')
       failure_message _('Could not upload Scap Contents')
 
+      def execute
+        d = send_request
+        d.delete('errors') if d['errors'].empty?
+        if d['results'].any? || d['errors'].any?
+          print_data(d)
+        else
+          print_success_message(d)
+        end
+        HammerCLI::EX_OK
```

Comment:
Is it still ok if `d['errors'].any?`?

Reply:
Well, sometimes they can be something that can be ignored? https://github.com/theforeman/foreman_openscap/blob/master/lib/foreman_openscap/bulk_upload.rb#L40

I mean, it seems that we could return both OK or ERR. If we return ERR, but some of the content was actually uploaded, it's kinda misleading if you don't read the output. But if you expect everything to be uploaded and expect OK, but you get OK even though not everything was uploaded, it's... still misleading, so in this case IDK :D

Reply:
I'd like us to be a bit more strict and go with ERR. If we're asked to upload something and we, for any reason, don't do that, I'd expect that to be not ok.

Reply:
Thanks, updated.